ncl/ncl_index/ncl-index-table.csv

```diff
@@ -1,3 +1,5 @@
 NCL Function, Description, Python Equivalent, Notes
 `days_in_month <https://www.ncl.ucar.edu/Document/Functions/Built-in/days_in_month.shtml>`__,"Calculates the number of days in a month given month and year","``cftime.datetime().daysinmonth``",`details <../ncl_entries/days_in_month.ipynb>`__
 `day_of_week <https://www.ncl.ucar.edu/Document/Functions/Built-in/day_of_week.shtml>`__,"Calculates the day of the week given month, day, and year","``cftime.datetime().dayofwk``",`details <../ncl_entries/day_of_week.ipynb>`__
+`specx_anal <https://www.ncl.ucar.edu/Document/Functions/Built-in/specx_anal.shtml>`__," Calculates spectra of a 1-D series.","``scipy.signal.periodogram()``, ``scipy.signal.convolve()``",`example notebook <../ncl_entries/specx_anal.ipynb>`__
+`specxy_anal <https://www.ncl.ucar.edu/Document/Functions/Built-in/specxy_anal.shtml>`__," Calculates spectra of a 2-D series.","``scipy.signal.periodogram()``, ``scipy.signal.convolve()``, ``scipy.signal.csd()`, ``scipy.signal.coherence()``",`example notebook <../ncl_entries/specxy_anal.ipynb>`__
```

Comment:
There are extra marks on `scipy.signal.csd()` and `scipy.signal.coherence()`
```
scipy.signal.periodogram(), scipy.signal.convolve(), scipy.signal.csd()`, ``scipy.signal.coherence()
```

Reply:
Good eye @cyschneck ironically the extra "`" appeared because I was missing one.